Stackage/Database/Cron.hs

```diff
@@ -222,16 +221,27 @@ createHoogleDB db man name = handleAny (\e -> print e $> Nothing) $ do
     void $ tryIO $ removeFile (fromString outname)
     createTree (fromString bindir)
 
-    dbs <- runResourceT
-        $ sourceTarFile False tarFP
-       $$ evalStateC 1 (mapMC (singleDB db name bindir))
-       =$ sinkList
-
-    putStrLn "Merging databases..."
-    Hoogle.mergeDatabase (catMaybes dbs) outname
-    putStrLn "Merge done"
-
-    return $ Just outname
+    withSystemTempDirectory ("hoogle-" ++ unpack (toPathPiece name)) $ \_tmpdir -> do
+        let tmpdir = "/Users/michael/Desktop/hoo"
```

Comment:
Just a reminder, you'll probably be updating this when you setup the cron on prod. @snoyberg